pydecorate/decorator_agg.py

```diff
@@ -30,8 +33,8 @@ def add_scale(self, colormap, **kwargs):
         self._add_scale(colormap, **kwargs)
 
     def _load_default_font(self):
-        font_path = resource_filename("pydecorate.fonts", "DejaVuSerif.ttf")
-        return aggdraw.Font("black", font_path, size=16)
+        font_path = HERE / "fonts" / "DejaVuSerif.ttf"
+        return aggdraw.Font("black", font_path.as_posix(), size=16)
```

Comment:
Aggdraw needs a posix-y path for this? I've never used aggdraw on Windows so I wasn't sure.

Reply:
It just needed a string. I imagine it should work with `str(font_path)` as well, I just prefer to see the `/` separators so usually use `.as_posix()` since most Windows things accept it (seems to work fine in the CI here).

Reply:
Ok. I don't feel strongly enough to change it. It does, at a glance, make a reader of the code think that Windows paths are supported. Let's merge this! Thanks.